### ISSUE TITLE
Disable logging to access log for a route defined in http.routers

### ISSUE THREAD

**Issue by jerrywoo96** [traefik/traefik#11144]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you expect to see?

I would like to disable logging for a route that is defined in http.routers. My use-case is that i have configured a DNS-over-HTTPS route to a dns server, and as you know dns requests are extremely frequent, they are written to logs and most of the content in the access logs are filled up by DNS-over-HTTPS requests.

![image](https://github.com/user-attachments/assets/eb42ca43-3926-4034-aedf-9842f9553a82)

I would like to disable logs for a specific router by setting an attribute to disable logs. An example of a configuration would be:

```yaml
http:
  routers:
    Route1:
      logs: false
```

Or another example might be to define which route can write to which access logs.

static:
```yaml
accessLog:
  Log1:
    asDefault: true # New Feature
    filePath: path/to/Log1.log
    addInternals: true
    fields: []
  Log2:
    filePath: path/to/Log2.log
    addInternals: true
    fields: []
```

dynamic:
```yaml
http:
  routers:
    Route1: # logs to Log1 only
      logs:
      - Log1
    Route2: # logs to Log2
      logs:
      - Log2
    Route3: # logs to Log1 and Log2
      logs: 
      - Log1
      - Log2
    Route4: # disables logging to any accessLogs
      logs: false
    Route5: # Use default log which is Log1
```

I only do require the function to disable logs, although logging to different log files would be a plus to filter out logs from a jumbled mass of logs as log files can get quite large.

If there is already an existing configuration that enables this function, do provide me with examples or suggestions. Thanks!

**Comment by mohitk-brevo** [comment]
Hello, 
Is this issue already in dev? I am interested in working on it.

**Comment by rtribotte** [comment]
@jerrywoo96 Indeed, PR https://github.com/traefik/traefik/pull/11308 has been merged and is addressing this issue, thank you for noticing this.
I'm closing this issue accordingly.
Closed by #11308
